vhdbuilder/packer/components.json

```diff
@@ -50,6 +53,9 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/cni-dropgz:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "v0.0.4.1",
+        "v0.0.11",
+        "v0.0.15",
```

Comment:
should be fine to not update prefetch optimizations for now I suppose, will probably need to come back to it after we finalize plans with the image builder folks tho